packages/material_tree/README.md

```diff
@@ -0,0 +1,206 @@
+# Material Tree
+
+Material Tree is an enrichment to React Tree component. 
+
+It is based on basic React Tree logic and allows 
+a user to display and manage a hierarchical structure of items using <b>Material-UI</b> components.
+
+ 
+  <p align="center">
+      <img src="react-tree-demo.gif?raw=true" width="400" />
+  </p>
+  
+  ### Installation
+  Since Material Rree is based on <b>React Tree</b> logic, you need to install both react-tree and material-tree.
+  
+  In addition, dependencies to <b>material-ui icons and core components</b> are needed if they don't exist in your project.
+  
+   **Installation using npm:**
+  ```
+   npm install @kenshooui/material-tree --save
+
+   npm install @kenshooui/react-tree --save
+
+   npm install @material-ui/core --save
+
+   npm install @material-ui/icons --save
```

Comment:
You can do this, and the yarn command in a single line - 
npm install --save @material-ui/core @material-ui/icons @kenshooui/react-tree @kenshooui/material-tree
yarn add @material-ui/core @material-ui/icons @kenshooui/react-tree @kenshooui/material-tree

Also worth mentioning that the material dependencies are only if they don't exist already in the app.

Reply:
@liorheber isn't this enough: "In addition, dependencies to <b>material-ui icons and core components</b> are needed if they don't exist in your project." ?
Do you want me to rephrase it?